Description: Add armhf and arm64 to the bootstrap targets, to allow building an instrumented compiler on those platforms too
Author: Olivier Tilloy <olivier.tilloy@canonical.com>

--- a/tools/clang/scripts/build.py
+++ b/tools/clang/scripts/build.py
@@ -673,9 +673,7 @@ def main():
       projects += ';libcxx;compiler-rt'
 
     bootstrap_targets = 'X86'
-    if sys.platform == 'darwin':
-      # Need ARM and AArch64 for building the ios clang_rt.
-      bootstrap_targets += ';ARM;AArch64'
+    bootstrap_targets += ';ARM;AArch64'
     bootstrap_args = base_cmake_args + [
         '-DLLVM_TARGETS_TO_BUILD=' + bootstrap_targets,
         '-DLLVM_ENABLE_PROJECTS=' + projects,
